src/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/SyndicationFeedFormatter.cs

```diff
@@ -40,6 +40,15 @@ public SyndicationFeed Feed
             }
         }
 
+        public Func<string, string, string, string> StringParser { get; set; } = DefaultStringParser;
+
+        public Func<string, UriKind, string, string, Uri> UriParser { get; set; } = DefaultUriParser;
+
+        // Different DateTimeParsers are needed for Atom and Rss so can't set inline
+        public Func<string, string, string, DateTimeOffset> DateTimeParser { get; set; }
+
+        internal abstract Func<string, string, string, DateTimeOffset> GetDefaultDateTimeParser();
+
```

Comment:
This will prevent anyone from being able to derive from the class as they won't be able to provide an implementation for this method. Make it virtual instead and have the base implementation do something sensible.

Reply:
Good point. I changed the method to virtual with default implementation.